fe/fe-core/src/main/java/com/starrocks/catalog/Dictionary.java

```diff
@@ -396,8 +423,6 @@ public List<String> getInfo() {
 
     @Override
     public void gsonPostProcess() throws IOException {
-        this.resetState();
-        this.updateNextSchedulableTime(this.getRefreshInterval());
     }
```

Comment:
remove this method and `implements GsonPostProcessable`

Reply:
Fixed